src/main/webapp/app/clinical/clinical.component.html

```diff
@@ -25,6 +24,23 @@
                     <ng-select class="selectStyle" [items]="subTypesOptions[clinicalInput.main_type]" [(ngModel)]="clinicalInput.sub_type"
                         [clearable]="true" placeholder="Select Subtype" (change)="onSingleSelected($event)">
                     </ng-select>
+                    <i class="collapse-margin" aria-hidden="true" title="Display optional attributes"
+                       ngClass="{{ displayOptionalFields ? 'fa fa-angle-double-up' : 'fa fa-angle-double-down'}}" (click)="collapse()"></i>
+                </div>
+            </div>
+        </div>
+        <div *ngIf="displayOptionalFields">
+            <div class="row">
+                <div class="col-sm-4">gender:</div>
```

Comment:
I would suggest you make gender a drop-down list with male and female, just to keep the interface consistency